docs/client-side-usecases configuration.md

```diff
@@ -0,0 +1,105 @@
+- `color`: `<Color>` a color object that defines the color of the use-case
+- `default`: `<Bool>` a `true|false` value that defines this use-case as the default one
+- `important`: `<Bool>` should this `use-case` a recommended one?
+
+For its first release, the manifest will be built-in the client app, to get it ready to work out-of-the-box. With the stack v2, we should serve the manifest behind a dedicated endpoint, so we won't need to build a specific version of _MyAccounts_ each time we need to provides different _use-cases_.
+
+
+Default values
+--------------
+
+There's two kind of default values for a _use-case_:
+
+### the `default` key
+
+it defines if this _use-case_ is the default one. When we access the _use-cases_ screen (`/discover` URI), it offers a list of _use-cases_ screen, each one redirecting to a _use-case_ view (or the konnector if the _use-case_ only contains one).
+
+Sometimes, we need to directly open a _use-case_ screen without passing by the `/discover` view (this is the case when user access to _MyAccounts_ from the onboarding). In this case, this is the default _use-case_ which is displayed.
+
+⚠️ if the _use-cases_ array contains more than one _use-case_ with a default key set to `true`, then only the first one find in the array is considered as the default one.
+
+### The `incentive`
+
+When displaying a _use-case_ screen, a konnector can be highlighted first to incitate the user to first configure this one. Into the `konnectors` array, the _Konnector Object_ can define a `default` key at `true` to declares it as the _incentive_ one.
+
+⚠️ as for the _use-cases_ array, if more than one konnector has a `default` key set to true, only the first one in the array is considered as the _incentive_ one.
+
+
+Localization
+------------
+
+There's 2 kind of information that need to be translated:
+
+1. the _use-case_ name (mandatory)
+2. a _use-case_ description (optionnal)
+
+To avoid having all translations for all use-cases embeded into the build app, even when the _use-cases_ aren't declared in the given context, all locales related to a context should belong to the context directly, so located in `client/app/contexts/<context>/locales/<lang>.json` file.
+
+It uses a syntax like `use-case <slug> title` and `use-case <slug> description` as translation keys. Transifex remains the centralized tool for all translations, contexts included.
+
+
+Architecture of the manifest
+----------------------------
+
+A manifest should have the following structure:
+
+```json
+{
+  "use-cases": [{
+    "slug": "the use-case slug",
+    "konnectors": [{
+      "slug": "konnector-to-add-slugname",
+      "default": true,
+      "important": true
+    }],
+    "figure": "use-case_picture_file_name.ext",
+    "color": {
+      "css": "the CSS value of the use-case color"
+    },
+    "default": true,
+    "important": true
+  }]
+}
+```
+
+The `important` key indicate if the _use-case_/_konnector_ is recommended or not.
```

Comment:
Those last two property descriptions seem like a repetition of what has been previously told int the document. What do you think ?